### PR TITLE
AddressTypeChange for CE Case ceActualResponses Defaulted to Zero

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
When an AddressTypeChange event occurs where the new address is a CE Case, the ceActualResponses is null rather than 0 

# What has changed
Changed ceActualResponses from Integer to int so now defaults to 0 instead of null
Test

# How to test?
Maven clean install on branch and check image builds and tests pass

# Links
https://trello.com/c/DUgevj8y

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[CE case missing ceActualResponses](https://trello.com/c/DUgevj8y/1494-ce-case-missing-ceactualresponses)

File: src/main/java/uk/gov/ons/census/casesvc/service/AddressTypeChangeService.java [hunk 1]
```diff
@@ -98,6 +98,10 @@ private void createNewCase(
     newCase.setHtcWillingness(oldCase.getHtcWillingness());
     newCase.setHtcDigital(oldCase.getHtcDigital());
 
+    if (newCase.getCaseType().equals("CE")) {
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/casesvc/service/AddressTypeChangeService.java:101]: Why only do this for CE cases? Are we taking the wrong approach completely. We could change `ceActualResponses` from an `Integer` to an `int` and save ourselves headache.

Could do the same with `ceExpectedCapacity` - make it an `int` and save ourselves a bunch of weird null pointer exceptions.

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
I think there's a better approach. See comments.

**Review by NickGrantONS — APPROVED** [review]
Looks good. I wonder if we're going to get null pointer exceptions when people start sending us new cases with null `ceActualResponses` but that's their problem, not ours

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**